packages/select/src/common/itemListRenderer.ts

```diff
@@ -57,7 +57,7 @@ export function renderFilteredItems(
     noResults?: React.ReactNode,
     initialContent?: React.ReactNode | null,
 ): React.ReactNode {
-    if (props.query.length === 0 && initialContent !== undefined) {
+    if (props.query.length === 0 && initialContent != null) {
```

Comment:
this change feels wrong -- the explicit undefined check is supported by the docs.